spec/drafts/v2.1.1/stix-v2.1.1.adoc

```diff
@@ -11435,7 +11420,7 @@ The threat actor type vocabulary is currently used in the following SDO(s):
 
 Threat actor type is an open vocabulary used to describe what type of threat actor the individual or group is.
 For example, some threat actors are competitors who try to steal information, while others are activists who act in support of a social or political cause.
-Actor types are not mutually exclusive: a threat actor can be both a disgruntled insider and a spy. <<Casey_2007>>
+Actor types are not mutually exclusive: a threat actor can be both a disgruntled insider and a sensationalist.
```

Comment:
Actor types are not mutually exclusive. A threat actor can have multiple motivations.

Reply:
`Actor types are not mutually exclusive. A threat actor can have multiple motivations.` seems indeed much better than the two examples from the original document.